.github/workflows/DOCER.yml

```
@@ -0,0 +1,67 @@
+
+on: pull_request
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.12)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)